linkml_runtime/utils/yamlutils.py

```diff
@@ -1,3 +1,4 @@
+import pdb
```

Comment:
This should not be here, right? It was only for you during debugging, right?

Reply:
right my b